### PR TITLE
feat(i18n): sync, add `ca` and link to translation project 

### PR DESCRIPTION
**Pull request by lidel**
This PR:

- synchronizes translations
- adds Catalan locale
- adds link to IPFS Translation Project at Transifex to the `LanguageModal`
  - should improve the discoverability for people who want to add/improve translations

### CODE REVIEW

File: src/components/language-selector/language-modal/LanguageModal.js [hunk 1]
```diff
@@ -28,6 +28,10 @@ const LanguageModal = ({ t, tReady, onLeave, link, className, ...props }) => {
             </button>
           )}
         </div>
+        <p className='lh-copy charcoal f6'>
+          {t('languageModal.translationProjectIntro')}<br/>
+          <a href="https://www.transifex.com/ipfs/public/" rel="noopener noreferrer" target="_blank" class="link blue">{t('languageModal.translationProjectLink')}</a>
+        </p>
```
Comment (lidel) [inline src/components/language-selector/language-modal/LanguageModal.js:34]: Feel free to tweak style / text so it looks better :pray: initial PoC looks like this:

File: public/locales/en/settings.json [hunk 2]
```diff
@@ -8,7 +8,9 @@
   "config": "IPFS Config",
   "languageModal": {
     "title": "Change language",
-    "description": "Pick your preferred language."
+    "description": "Pick your preferred language.",
+    "translationProjectIntro": "Missing your language? Want to improve an existing one?",
```
Comment (jessicaschilling) [inline public/locales/en/settings.json:12]:
```suggestion
    "translationProjectIntro": "Add or improve translations and make IPFS better for everyone!",
```

File: public/locales/en/settings.json [hunk 3]
```diff
@@ -8,7 +8,9 @@
...
+    "translationProjectLink": "Join IPFS Translation Project"
```
Comment (jessicaschilling) [inline public/locales/en/settings.json:13]:
```suggestion
    "translationProjectLink": "Join the IPFS Translation Project"
```

**Comment by lidel** [comment]
@jessicaschilling done, now looks like this:

> Screenshot_2020-05-28 Settings - IPFS


Let me know if it is ok to merge.

FYSA I'd like to create a new webui release with recent fixes we all did (including this one) and try to ship it with go-ipfs 0.6 so more users can benefit from this :pray:

**Comment by lidel** [comment]
We also may need to reconsider style for links overall 
(`link blue` turns teal-ish after link is visited)
but thats something for future PRs :-) merging!

**Pull request merged**